src/internal/internal_trsmA.cc

```diff
@@ -311,35 +320,35 @@ void trsmA(internal::TargetType<Target::Devices>,
 
                     blas::Queue* queue = A.compute_queue( device, queue_index );
                     assert( queue != nullptr );
+                    queue->sync();
```

Comment:
Did adding a sync fix something?

Reply:
That was from debugging something and was suppose to be removed.  I'll remove it.